Cargo.toml

```diff
@@ -31,4 +31,6 @@ near-primitives = "0.21.0"
 near-crypto = "0.21.0"
 derive_builder = "0.20.0"
 clap = { version = "4.5", features = ["derive"] }
-dotenv = { version = "0.15" }
\ No newline at end of file
+dotenv = { version = "0.15" }
+tracing = "0.1"
```

Comment:
Why you chose this lib?

Reply:
I think it was discussed within the team that we prefer `tracing` to `log`